service/controller/clusterapi/v29/resource/tccpsubnet/resource.go

```diff
@@ -81,6 +82,12 @@ func (r *Resource) addSubnetsToContext(ctx context.Context) error {
 						aws.String(cc.Status.TenantCluster.TCCP.VPC.ID),
 					},
 				},
+				{
+					Name: aws.String(key.TagTCCP),
```

Comment:
Filter out only control plane subnets from VPC (node pools also have subnets but they are part of separate CloudFormation stack & controller)

Reply:
Same worry here. Are we sure this can simply be changed? What is `cc.Status.TenantCluster.TCCP.Subnets` used for?

Reply:
Actually not. I also realized this filtering need to be moved to that mapping function above. These are also used for IPAM. I'll change this.